src/components/etherpad.jsx

```diff
@@ -12,23 +12,23 @@ import {getFilters} from '../route-utils';
 import Loadable from './loadable';
 import GithubFlavoredMarkdown from './gfm';
 
-const EtherpadInner = React.createClass({
-  getDefaultProps() {
-    return {
-      hostName: 'https://openstax-pad.herokuapp.com',
-      secret: 'openstax'
-    };
-  },
-  getInitialState() {
-    return {isSaving: false, text: 'Please wait, it may take 30sec for the free Heroku site to spin up. See [Heroku Free Dynos](https://blog.heroku.com/archives/2015/5/7/heroku-free-dynos)'};
-  },
+class EtherpadInner extends Component {
+  static defaultProps = {
+    hostName: 'https://openstax-pad.herokuapp.com',
```

Comment:
We shouldnt modernise a component that we are not able to test

https://github.com/coala/gh-board/issues/20

Reply:
React.createClass is not supported in React V16.
A feasible approach is to `import createReactClass from "create-react-class";` and substitute `React.createClass` with `createReactClass`.